### DISCUSSION TITLE
Electrum pull payment

### DISCUSSION THREAD

**Discussion by cryptokq** [btcpayserver/btcpayserver#2982] [Technical Support]
I am trying to make a test pullpayment using electrum 4.0.2 seed but I am getting the following error:

The master fingerprint and/or account key path of your seed are not set in the wallet settings.

Where do I get the master finger print or account key path for electrum?

**Comment by bolatovumar** [comment]
@cryptokq idk if this helps you but BTCPay specifies the internal wallet key path Under "Wallets" -> "Manage" -> "Settings".
